src/extension.ts

```diff
@@ -1,80 +1,119 @@
-import * as vscode from 'vscode'
-const modeConfigurationKey = 'syncScroll.mode'
-const contributionPoints = require('../package.json').contributes
-const [toggleCommand, changeModeCommand] = contributionPoints.commands
-const { [modeConfigurationKey]: modeConfiguration } = contributionPoints.configuration.properties
+import * as vscode from 'vscode';
```

Comment:
I think you should remove every `;` to follow the style in this repository (not using `;`) so this PR merges seamlessly @k-ta-yamada